src/5e-SRD-Monsters.json

```diff
@@ -21,29 +21,34 @@
     "charisma": 18,
     "proficiencies": [
       {
+        "index": "saving-throw-con",
         "name": "Saving Throw: CON",
-        "url": "/api/proficiencies/saving-throw-con",
-        "value": 6
```

Comment:
Also, not in this PR, but I'm not sure how I feel about having the value in here.

Reply:
Again, see the original PR comment.